patterns/page-home-business.php

```diff
@@ -14,5 +14,61 @@
 <!-- wp:pattern {"slug":"twentytwentyfour/text-feature-grid-3-col"} /-->
 <!-- wp:pattern {"slug":"twentytwentyfour/text-alternating-images"} /-->
 <!-- wp:pattern {"slug":"twentytwentyfour/testimonial-centered"} /-->
-<!-- wp:pattern {"slug":"twentytwentyfour/posts-grid-2-col"} /-->
+
```

Comment:
We should still reference this new pattern properly as a pattern; to set the standard for theme creators looking at this in the future. We're not adding a changed i18n string; it's the same string as in the posts-grid-2-col pattern.

Reply:
I think it's fine to leave the old pattern; we just don't use it in a template but it would still be available to insert wherever?

Reply:
Yeah, IMO it would be fine to keep the old pattern.

On another note, do we want to move the new markup below into a new pattern so that it is reusable, or should it remain only as part of this greater `page-home-business.php` pattern?

Reply:
Apologies, maybe I didn't understand the discussion fully. Is the concern about introducing new translation strings? If so, I personally don't think we should have that limit ourselves from making this a reusable pattern, but I'm aware of the late translation string changes in the release cycle not being preferable. But maybe this is an exception worth making?

Let me just say: If introducing a new translation string for the new pattern title isn't a concern, IMO it should be a new pattern, including a new title. If we want to avoid a new translation string, then I guess it's okay to keep this as is.

Reply:
I honestly don't know how much of a leeway we have when introducing new strings after freeze! but that is definitely the only thing stopping us from making a pattern here. If you think this is ok to do, then I say go for it, let's add a pattern

Reply:
I would be in support a new pattern here.